R/deprecated.R

```diff
@@ -89,7 +89,7 @@ memoryCache <- function(
   exec_missing = FALSE,
   logfile = NULL)
 {
-  shinyDeprecated("cachem::cache_mem", version = "1.5.1")
+  shinyDeprecated("1.5.1", "diskCache()", "cachem::cache_mem()")
```

Comment:
```suggestion
  shinyDeprecated("1.6.0", "diskCache()", "cachem::cache_mem()")
```

This also should have a check for `exec_missing` like above.